tests/workflows/test_consistency.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+import zipfile
+from datetime import datetime
+from multiprocessing import cpu_count
+from pathlib import Path
+
+import pytest
+from fmriprep import config
+from halfpipe.cli.parser import build_parser
+from halfpipe.cli.run import run_stage_run
+from halfpipe.file_index.bids import BIDSIndex
+from halfpipe.model.spec import save_spec
+from halfpipe.workflows.base import init_workflow
+from halfpipe.workflows.execgraph import init_execgraph
+
+from .datasets import Dataset, datasets
+from .spec import TestSetting, make_spec
+
+settings_list: list[TestSetting] = [
+    TestSetting(
+        name="FalseComb0",
```

Comment:
```suggestion
        name="noCounfounds",
```
I think it would be great to give the settings human-readable names

Reply:
I would like to keep a commit for traceability with the names that we used to get data with this version. 

I think we will miss the overview with the change of names if we squash this branch into one commit and I would prefer to not make it dependant to having on local file or other solution with the reference, but rather keep the timeline in the github page where it is accessible by everyone and we can see the changes.

I will change to more human-readable names after, when we start testing with the updated fmriprep and also create the new github action.

Reply:
Sounds good! You could even add a comment with the previous names, or just a dictionary mapping from old names to new names, that we could directly use in future code

Reply:
Just updated my previous comment after re-reading your message :-)

Reply:
Perfect, see my last commits for the new names